info-commands.go

```diff
@@ -275,7 +275,7 @@ func (info InfoMessage) StandardParity() int {
 
 // Services contains different services information
 type Services struct {
-	KMS           KMS                           `json:"kms,omitempty"`
+	KMS           []KMS                         `json:"kms,omitempty"`
```

Comment:
This can break the compabitility of newer mc with an older MinIO server, can we add a new fields instead and add comment for KMS field (Deprecated July 2023)

Reply:
Sure. Thanks for suggestions. Will update it.